### ISSUE TITLE
Update README.rst

### ISSUE THREAD

**Issue by toxinu** [toxinu/leselys#113]
- [ ] Screenshots 404
- [ ] RabbitMQ install
- [ ] Create virtualenv without creating directory
- [ ] Missing deps (zlib1g-dev) for at least Ubuntu 14.04

**Comment by jantman** [comment]
Heroku demo app doesn't seem to be working either.

**Comment by socketubs** [comment]
@jantman Yes, since a long time, because I don't maintain Leselys anymore. But you can of course set up it very easily to test test :smile:
